src/bidiMapper/BidiServer.ts

```diff
@@ -36,8 +36,8 @@ export class BidiServer extends EventEmitter<BidiServerEvents> {
   #messageQueue: ProcessingQueue<OutgoingBidiMessage>;
   #transport: BidiTransport;
   #commandProcessor: CommandProcessor;
-  #browsingContextStorage: BrowsingContextStorage;
-  #realmStorage: RealmStorage;
```

Comment:
Please refrain from reformat when doing functional changes unless it is required, as it can mask some issues.

Reply:
Those 2 classes need to be initialize before all, as they are probed on some Class creation and result in errors if the order it wrong this insures the order is always correct.